### DISCUSSION TITLE
Kraken as “preferred” not working

### DISCUSSION THREAD

**Discussion by ddman2** [riffnshred/nhl-led-scoreboard#299] [Dev and Ideas]
Trying to set up a board with Kraken as the preferred team.  Other teams work, but if I set the team to “Kraken” it gets stuck at “Loading…”

**Comment by pjockey** [comment]
I guessing the software needs to be updated to contain the Kraken.

**Comment by riffnshred** [comment]
Thats normal for now. I won't fix it until the pre-season start incase things change in the API and I have to do fixes related to the Kraken joining the league again. I might have something in the beta in the next few weeks though. I'll keep you guys posted.
